databroker/_core.py

```diff
@@ -2063,7 +2070,7 @@ class Broker(BrokerES):
         to do all registration manually.
 
     """
-    def __init__(self, mds, reg=None, plugins=None, filters=None,
+    def __init__(self, mds, reg=None, plugins=None, filters=None,  archivers = None,
```

Comment:
We should definitely not add an `archivers` keyword argument to `Broker`. The Archiver should be "just another data source".